src/foam/nanos/theme/Theme.js

```diff
@@ -100,6 +100,23 @@ foam.CLASS({
       name: 'appName',
       section: 'infoSection',
     },
+    {
+      class: 'String',
+      name: 'loginDAO'
+    },
+    {
+      class: 'String',
```

Comment:
```suggestion
      class: 'Image',
```
Also add a dualView with a textbox view, look elsewhere in Theme,  so there is somewhere to add the byte encoded image, and the dual view to view it.

Reply:
Okay, will do.